src/main/java/org/team1540/robot2024/RobotContainer.java

```diff
@@ -214,17 +215,24 @@ private void configureButtonBindings() {
 
         new Trigger(indexer::isNoteStaged).and(intakeCommand::isScheduled).onTrue(CommandUtils.rumbleCommandTimed(driver.getHID(), 0.8, 0.4));
 
-        new Trigger(RobotController::getUserButton).toggleOnTrue(Commands.startEnd(
-                () -> {
-                    elevator.setBrakeMode(false);
-                    leds.setPatternAll(() -> new LedPatternRSLState(Color.kMagenta), Leds.PatternLevel.ELEVATOR_STATE);
-                },
+
+        BooleanSupplier isPreMatch = () -> (!DriverStation.isDSAttached() || DriverStation.isAutonomous()) && DriverStation.isDisabled();
+        Command brakeModeCommand = Commands.runOnce(
                 () -> {
-                    elevator.setBrakeMode(true);
-                    leds.clearPatternAll(Leds.PatternLevel.ELEVATOR_STATE);
+                    if (!isPreMatch.getAsBoolean()) {return;}
+                    if (!isBrakeMode) {
+                        enableBrakeMode(true);
+                    } else {
+                        disableBrakeMode();
+                    }
                 }
-        ).ignoringDisable(true));
+        ).ignoringDisable(true);
+
+        new Trigger(tramp::isNoteStaged).debounce(0.1)
+                .onTrue(brakeModeCommand);
```

Comment:
Instead of checking isPreMatch in brakeModeCommand it would probably be less sketchy if you just chained it to the trigger with a `.and()`

Reply:
That's how I had it originally, but then a change in disabled state triggers the command, which is not what I want